login/profile.go

```diff
@@ -15,6 +15,7 @@ import (
 const ImageURLAttributeName = "ImageURL"
 const BioAttributeName = "Bio"
 const URLAttributeName = "URL"
+const CompanyAttributeName = "Company"
```

Comment:
Attribute name should be low case - "company". Attribute names are case sensitive!  And we are using "company" in KC. So your PR creates a new one "Company" instead of re-using the existing "company".
And I think we should use low case names for all attributes such as bio, url, imageURL, etc for constancy with api payloads names and with "company" in KC.